app/views/groups/index.html.erb

```diff
@@ -26,7 +26,11 @@
 						<%= link_to group, :class => 'titgrcard' do%>
 							<%= group.name %>
 						<% end %>
-						<div class="subtgr"><b><%= t('biomodelos.groups.index.moderator') %></b></br><%= group.admin.first.user.name %></div>
+						<div class="subtgr">
+                            <b><%= t('biomodelos.groups.index.moderator') %></b>
+                            </br>
+                            <%= group.admin.first ? group.admin.first.user.name : t('biomodelos.groups.index.no_moderator') %>
+                        </div>
```

Comment:
Fix the indentation. Since it's mixed between tabs and spaces it looks bad. Either change the tab size to 2 in your editor and reduce the indentation for these lines or change the whole file to spaces.

Reply:
Fixed